server/core/storage.go

```diff
@@ -52,25 +54,26 @@ const (
 // Storage wraps all kv operations, keep it stateless.
 type Storage struct {
 	kv.Base
-	regionStorage    *RegionStorage
-	useRegionStorage int32
-	regionLoaded     int32
-	mu               sync.Mutex
+	regionStorage        *RegionStorage
+	encryptionKeyManager *encryptionkm.KeyManager
+	useRegionStorage     int32
+	regionLoaded         int32
+	mu                   sync.Mutex
 }
 
 // NewStorage creates Storage instance with Base.
-func NewStorage(base kv.Base) *Storage {
+func NewStorage(
```

Comment:
How about using `opts ...Option` code design? This is a method compatible with the old code.